rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, person, book)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ]  just as previous reviewer said Kindly make sure that the rental is adding itself to the @book achieving the 'belongs-to' association
`book.rentals << self `and the rental is adding itself to the @person the 'belongs-to' association
 `person.rentals << self ` kindly add those relations by add those to short code  :+1: